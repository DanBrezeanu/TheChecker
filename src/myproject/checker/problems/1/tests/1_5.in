70116384
208450936
757288745
-411977633
17830336
997013450
-814346769
972315297
983792363
245668656
-133663189
172931814
-234146339
-14673203
-599397334
585469703
652199232
134241016
-151868713
-194988326
-361689232
-803145075
842467989
-420098268
-959642957
-500537121
-500628650
982042882
628606917
553745025
865565633
-935893252
796411712
953974549
410503232
839843619
850402813
-388778155
-896715014
-525285847
553632545
466901675
-8703058
314325121
40589500
-17632860
-400562687
542484030
312431277
-880017980
238786506
-626847509
-880872943
788145764
-105415245
17271907
-175318118
-201099877
-283050774
623787913
762976182
-858279956
-859139679
471308036
-519764649
-404559962
287794642
-143820005
-737285244
-534936968
971434294
-277427179
-268377633
-325060441
323023790
-925729456
-118269241
-659612497
161274628
-257441649
398365675
760871393
592405653
13756142
966808806
133239490
-489295435
182789887
-474918391
-165415177
857123489
54970917
133590909
-733215309
-706075904
-385625555
-295505523
-775624133
-899166427
578327066
714769486
968268227
-89821304
989497910
842240205
-386508134
815848288
-340345597
-327392790
261678066
706637582
-494123362
213591407
819775561
920572425
-473285512
-598638938
966307089
296957925
451857880
339629266
526983851
631674471
-965264048
393173410
275755731
132077425
-390853022
118267670
108492957
-67595320
-655644674
-791556329
39558875
-397383407
243113631
-162151102
-585693641
508104247
525668380
455475642
294327864
-511965270
759386994
432417025
106341175
527829319
-27009385
-627157560
-430457458
477417756
-961113038
-774201566
-73746130
998021363
-83076005
838631259
743804678
767182200
-785965734
-463114032
-670838483
-872321060
253420625
30091203
-891733246
-230476551
873105194
-706426538
818591035
33942045
-260007050
-176659544
903138899
963367676
-484180784
-77042744
-906859277
-479750127
-297951945
512044502
-217843083
875525539
-869710579
-938538791
655449055
331622269
701490814
-889913411
-995773381
782464028
-941197721
564655455
839590700
-723615136
232999478
38707459
-797894503
989999944
427548573
-207860218
728367896
282159030
366895429
-866529841
-243258040
-592968426
661071822
552052108
348176965
-682582736
584128404
-90381551
-29640953
-549244657
-809707367
-56576614
-731881169
987586154
-324593701
356473339
661338912
-675433998
729188762
-276472183
-814259716
858287654
393158186
665821642
-59175252
-381697266
30288139
31668188
122597653
-888740485
463842018
-822127376
-554670852
154461030
328721468
166370680
-67789018
-18864878
-207097441
-417508514
-533548280
690393382
-93689242
-587976891
155768985
907769538
564096847
245493733
356640481
-173580067
-382128157
-940482854
610472581
-981502842
-32705050
255228256
569309181
481914468
-925023082
-81060015
250397866
-784794189
-834892615
-201174171
3181233
511590541
427430467
-511256257
594746844
939563195
850326530
433415389
658984693
-438706692
297243567
205837208
-114299680
-500109017
542630245
-287602109
32028346
-44963490
-40812350
586924713
-582897723
434352715
834042543
213728355
172781479
288621320
870910758
-690970140
-417813393
-893526496
128025767
-291616717
-162936249
-666285714
-582292416
936953862
102937585
89241330
-257463633
746282322
776291813
838568531
-833770809
-884838521
-683404714
-317372660
-319613606
355270536
-415293908
-381565336
-55511266
-24808973
941774754
266593046
-974952803
-548840965
206235678
-680878943
462472197
-819564480
879482487
-545872071
660893521
-934252103
-467044075
-25341093
-867012942
924046094
81041939
-769202094
373427050
-211892887
-272861195
-218898652
93502017
-76931288
300744874
-100794547
415326649
127710060
-169124756
-239623655
364422514
-976937801
831490084
228957591
867863642
-762283317
-520400338
-750380694
728341323
-434051447
256047237
-63173554
-899891044
-799128402
429215698
539584617
534871345
-843576720
-303018258
-196058358
-293971956
-417807483
925299572
-598816683
785533980
-118407733
565266564
-135562283
813556360
-771419713
135206033
-947570742
-349759957
184727088
-389526298
734661850
-439440669
-123295619
686817088
56422616
-331555454
-97068255
699273224
997295867
-903358247
990991318
-385741802
623413084
357856311
320367941
-385110479
564887401
613817456
453557060
-347008644
23916469
-487851579
718858714
412908226
-582843388
247909257
-124754853
-714943317
-932659683
-267433711
533597517
841391017
837432926
770851004
-45861171
911898798
-634328074
826303391
377153837
-9151478
-28238883
-996977385
852397368
633610085
-939301457
-437636256
-960540609
-145531340
-451598634
610196258
-403452565
696899150
-250323391
624638611
707583821
-424964185
558199626
-521931927
0
