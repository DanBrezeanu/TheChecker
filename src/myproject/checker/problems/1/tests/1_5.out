-996977385
998021363
